app/controllers/api_controller/authentication.rb

```diff
@@ -42,6 +43,7 @@ def require_api_user_or_token
           @auth_user     = user.userid
           @auth_user_obj = userid_to_userobj(@auth_user)
           User.current_user = @auth_user_obj
+          authorize_user_group(@auth_user_obj)
```

Comment:
could you do this before `User.current_user`?
Feel it is cleaner if we get `@auth_user_obj` fully squared away before we assign it into `User`.

Current implementation won't cause issues. but not sure if we are going to start caching the tenant into another variable or something like that.